CefSharp.Core/OsrImeHandler.h

```diff
@@ -0,0 +1,117 @@
+// Copyright 2016 The Chromium Embedded Framework Authors. Portions copyright
+// 2013 The Chromium Authors. All rights reserved. Use of this source code is
+// governed by a BSD-style license that can be found in the LICENSE file.
+
+#ifndef CEF_TESTS_CEFCLIENT_BROWSER_OSR_IME_HANDLER_WIN_H_
+#define CEF_TESTS_CEFCLIENT_BROWSER_OSR_IME_HANDLER_WIN_H_
+#pragma once
+
+#include "Stdafx.h"
+#include <windows.h>
+//#include <vector>
+
+//#include "include/internal/cef_types_wrappers.h"
+
+namespace CefSharp {
+
+	// Handles IME for the native parent window that hosts an off-screen browser.
+	// This object is only accessed on the CEF UI thread.
+	class OsrImeHandler {
```

Comment:
Unless there is a specific reason this is a separate class I think this should be merged into `OsrImeWin`